nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -140,11 +211,11 @@ def _future_wait(self, future, timeout):
                 last_progress = current_progress
         return True
 
-    def send_request(self, channel, target, topic, request, timeout=10.0, secure=False, optional=False):
-
-        self.logger.debug(f"send_request: {channel=}, {topic=}, {target=}, {timeout=}")
+    def _send_request(self, channel, target, topic, request, timeout=10.0, secure=False, optional=False):
 
-        if channel != CellChannel.SERVER_COMMAND:
+        self.logger.info(f"send_request: {channel=}, {topic=}, {target=}, {timeout=}")
+        if channel not in CHANNELS_TO_HANDLE:
```

Comment:
Do you still need this check here? I thought it's already checked by the __getattr__ method.

Reply:
Thanks.  Forgot to remove it.  Fixed in the new commit.